### PR TITLE
Ato 626 porting

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- Moving the commits from PR https://github.com/RasaHQ/rasa/pull/11902/commits into 3.4.x.

### CODE REVIEW

File: .github/workflows/continous-integration.yml [hunk 1]
```diff
@@ -1046,4 +1046,4 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_TOKEN }}
         uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a
         with:
-          args: "⛔️ *Rasa Open Source* version `${{ github.ref_name }}` could not be released 😱! Please check out GitHub Actions: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          args: "⛔️ *Rasa Open Source* version `${{ github.ref_name }}` could not be released 😱! Please check out GitHub Actions: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```
Comment (ancalita) [inline .github/workflows/continous-integration.yml:1049]: Missing new line.

File: rasa/core/actions/action.py [hunk 2]
```diff
@@ -738,8 +738,11 @@ async def run(
             logger.debug(
                 "Calling action endpoint to run action '{}'.".format(self.name())
             )
-            response: Any = await self.action_endpoint.request(
-                json=json_body, method="post", timeout=DEFAULT_REQUEST_TIMEOUT
+            response = await self.action_endpoint.request(
```
Comment (ancalita) [inline rasa/core/actions/action.py:741]: This var needs the type hint back `Any` so that the code quality mypy check passes.

**Review by ancalita — APPROVED** [review]
Two minor details need to be sorted out before merging 👍🏻

**Comment by ancalita** [comment]
@sanchariGr I would hold off from merging this PR until Monday because of this [thread](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1674650314996349).

**Comment by sanchariGr** [comment]
> @sanchariGr I would hold off from merging this PR until Monday because of this [thread](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1674650314996349).

Ah thanks for pointing me to the thread @ancalita , i'll hold off.

**Review by radovanZRasa — COMMENTED** [review]
Postponing this PR until we resolve discussion on https://github.com/RasaHQ/rasa/pull/11953#pullrequestreview-1277079251

**Pull request closed without merging**